sdk/tests/builtins/logic_units/openapi_helper_files/petstore.json

```diff
@@ -0,0 +1,296 @@
+{
```

Comment:
is this available online via rest endpoint anywhere?

Reply:
No. Well it is but the site is broken. Doing it as a file is more reliable anyway